### DISCUSSION TITLE
Must I have a team to make projects?

### DISCUSSION THREAD

**Discussion by ketamine0389** [vercel/community#4153] [Account Issue]
### Question

I just signed up for Vercel through my Github account, and it says that I have a team account and it requires a teamId queryParam? As far as I know I should be under the default hobby account but it's doing nothing but prompt me to upgrade to pro. How do I fix this?

**Comment by amyegan** [comment]
Hey @ketamine0389. I don't see a team associated with your GitHub username. Some features are only available to Pro and Enterprise teams and will prompt you to upgrade if you attempt to enable them on a Hobby project or account. 

Can you tell me more about where you see the prompt or share some screenshots to help me see where this is happening?
